.dockerignore

```diff
@@ -1,3 +1,21 @@
-.git
-.env
-.dockerignore
+/.git
+/.github
+/log/*
+/tmp/*
+/tmp/pids/*
+!/log/.keep
+!/tmp/.keep
+!/tmp/pids/
+!/tmp/pids/.keep
+/config/master.key
+/.env
+/.DS_STORE
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
+/yarn-debug.log*
+/.yarn-integrity
+/app/assets/builds/*
+!/app/assets/builds/.keep
+/docker-compose*
```

Comment:
probably should ignore config/database files also so they aren't baked into the image